website/babel.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  presets: [
+    require.resolve('@docusaurus/core/lib/babel/preset'),
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current'
+        }
+      }
+    ]
+  ],
+};
```

Comment:
If we try using SWC, we might no longer need this file, and need to figure out how to [apply a similar config to what Docusaurus is doing](https://github.com/facebook/docusaurus/pull/6944/files#diff-bc4913d026ae51bef2bc8b7a1c040fdf52b999d99e6fa40d87b379f98a60e0ea).